scienzati_bot.py

```diff
@@ -4,14 +4,15 @@
 import sqlite3
 import random
 import re
+import sys
 
 #Settings class
 
 class Settings:
 	TelegramApiKey = "676490981:AAELlmTlQLD4_1HojhzWIX4yISDrVU5qDmA"
-	SupremeAdmins = [14092073]
-	ITGroup = 0
-	OTGroup = -1001176680738
+	SupremeAdmins = ["pandry","andreaidini"]#Lowercase username!
```

Comment:
by lowercase username is very very dangerous. Correct with telegramID (you have it in DB)

Reply:
Why is it considered to be dangerous?
another user can't take your username and you can easily undestand who the admins are (in the past ID were used)